prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -285,4 +305,41 @@ public void verify() {
             }
         }
     }
+
+    private void getListOfProfiles() throws Exception {
+        console.println(CliMessages.MESSAGES.availableProfiles());
+        Set<String> featurePackNames = KnownFeaturePacks.getNames();
+        final ChannelPrinter channelPrinter = new ChannelPrinter(console);
+        for (String profile : featurePackNames){
```

Comment:
Can you add some indentation so the output is a bit more structured? Something like this:

```
Profile: wildfly
   Subscribed channels:
      # wildfly
         manifest: org.wildfly.channels:wildfly
         repositories:
           id: central
           url: https://repo1.maven.org/maven2/
           id: jboss-public
           url: https://repository.jboss.org/nexus/content/groups/public/
           id: mrrc
           url: https://maven.repository.redhat.com/ga/
    Installed feature packs:
      org.wildfly:wildfly-galleon-pack:zip
```